pages/pipelines/hosted_agents/linux.md

```diff
@@ -28,26 +28,29 @@ To configure your Linux instance you can use the [Docker Compose](https://github
 
 ## Cache volumes
 
-Cache Volumes are external volumes attached to hosted agent instances. Cache Volumes are attached on a best-effort basis depending on their locality, expiration and current usage (so they should not be relied upon as durable data storage).
+_Cache volumes_ are external volumes attached to hosted agent instances. These volumes are attached on a best-effort basis depending on their locality, expiration and current usage, and therefore, should not be relied upon as durable data storage.
 
-Cache Volumes are disabled by default and can be enabled by providing a list of paths to cache at the pipeline level or the step level. Cache Volumes are scoped to a pipeline and are shared between all steps in a pipeline by default.
+By default, cache volumes:
 
-Cache Volumes act as regular disks with the following properties:
+- Are disabled, although you can enable them by providing a list of paths to cache at the pipeline- or step-level.
+- Are scoped to a pipeline and are shared between all steps in the pipeline.
 
-- They're backed by local NVMe storage. You can expect high performance.
-- A Cache Volume is formatted as a regular Linux filesystem (e.g. ext4), so you can expect them to support any use-case you have that Linux supports.
+Cache volumes act as regular disks with the following properties:
+
+- The volumes backed by local NVMe storage, delivering high performance.
+- The volumes are formatted as a regular Linux filesystem (e.g. ext4)—therefore, these volumes support any Linux use-cases.
 
 ### Cache configuration
 
-Cache paths can be defined in your [pipeline.yml](/docs/pipelines/defining-steps) file. Defining cache paths for a step will implicitly create a Cache Volume for the pipeline.
+Cache paths can be [defined in your `pipeline.yml`](/docs/pipelines/defining-steps) file. Defining cache paths for a step will implicitly create a cache volume for the pipeline.
 
-Defining cache paths enables mounting of the Cache Volume under `/cache` in the agent instance. The agent links subdirectories of the Cache Volume into the paths specified in the configuration. For example, defining `cache: "node_modules"` the agent will link `./node_modules` to `/cache/bkcache/node_modules`.
+When cache paths are defined, the cache volume is mounted under `/cache` in the agent instance. The agent links subdirectories of the cache volume into the paths specified in the configuration. For example, defining `cache: "node_modules"` in your `pipeline.yml` file will link `./node_modules` to `/cache/bkcache/node_modules` in your agent instance.
```

Comment:
Where does the intervening `/bkcache` directory come from?

I would have thought this would be `/cache/node_modules`?

Reply:
Currently by design.

Reply:
@matthewborden , quick question, would it be better to rewrite the first sentence to:

> When cache paths are defined, the cache volume is mounted under `/cache/bkcache` in the agent instance.

?

Reply:
`/cache/bkcache` should be relatively transparent to the customer, they should see that the volume is mounted into the directories they defined.

It would be more accurate to say:

> When cache paths are defined, the cache volume is mounted under /cache in the agent instance.

The `/bkcache` is created by the job setup scripts which will create matching directories for all the paths defined in `cache_paths` under `/cache/bkcache/user_defined_cache_path` then mount that directory into ./user_defined_cache_path within the checked out repo.